Gemfile

```diff
@@ -82,5 +82,6 @@ group :test do
   gem "faker"
   gem "rails-controller-testing", ">= 1.0.5"
   gem "shoulda-matchers", "~> 4.4"
+  gem "simplecov"
```

Comment:
ooh, are we adding coverage? are we gonna add some rules on 'not decreasing test coverage' or something similar?

Reply:
I wasn't planning to. We could, but I just like to have the info while writing tests to make sure I don't overlook something obvious